modules/op-uninstalling-the-pipelines-operator.adoc

```diff
@@ -6,10 +6,18 @@
 [id='op-uninstalling-the-pipelines-operator_{context}']
 = Uninstalling the {pipelines-title} Operator
 
+You can uninstall the {pipelines-title} Operator in web console.
+
+[CAUTION]
+====
+Uninstalling only the Operator does not remove the {pipelines-shortname} default and optional components. If you uninstall the Operator without removing the CRs of optional components, you cannot remove them later.
+====
+
 [discrete]
 .Procedure
-. From the *Operators* -> *OperatorHub* page, use the *Filter by keyword* box to search for `{pipelines-title} Operator`.
 
-. Click the *{pipelines-shortname} Operator* tile. The Operator tile indicates it is installed.
+. From the *Operators* -> *OperatorHub* page, use the *Filter by keyword* box to search for the {pipelines-title} Operator.
```

Comment:
Is it your convention to remove the special formatting for operator name here? It's OK if you are trying to refer to the product name but if the user is expected to type in this exact string, then the word `Operator` needs to be removed, otherwise the UI doesn't show any results. Maybe it would make sense to format it the same way as in following steps.

![olm](https://user-images.githubusercontent.com/445100/234533903-d5af2c9c-a7e4-4c37-8501-4bec70c261b2.png)

Reply:
@ppitonak I did not catch this. Thanks!